Apps/AdminWebClient/src/ClientApp/src/components/core/modals/BannerModal.vue

```diff
@@ -162,6 +183,21 @@ export default class BannerModal extends Vue {
     @Prop() editedItem!: Communication;
     @Prop() isNew!: number;
 
+    private get isDraft(): boolean {
```

Comment:
I meant the whole function can be simplified as 
```
 private get isDraft(): boolean {
            return this.editedItem.communicationStatusCode === CommunicationStatus.Draft;
    }
```

Reply:
ah how could I miss this :-)